From rdunlap@xenotime.net Sat Oct 15 22:08:45 2005
Date: Sat, 15 Oct 2005 22:03:30 -0700
From: "Randy.Dunlap" <rdunlap@xenotime.net>
To: linux-usb-devel@lists.sf.net
Cc: gregkh <greg@kroah.com>
Subject: [PATCH] usb doc: fix kernel-doc warning
Message-Id: <20051015220330.0997204a.rdunlap@xenotime.net>

From: Randy Dunlap <rdunlap@xenotime.net>

usb/core/buffer.c doesn't export any symbols, so it should use
!I instead of !E to eliminate this warning message:

Warning(/var/linsrc/linux-2614-rc4//drivers/usb/core/buffer.c): no structured comments found

Signed-off-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 Documentation/DocBook/usb.tmpl |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/Documentation/DocBook/usb.tmpl
+++ gregkh-2.6/Documentation/DocBook/usb.tmpl
@@ -291,7 +291,7 @@
 
 !Edrivers/usb/core/hcd.c
 !Edrivers/usb/core/hcd-pci.c
-!Edrivers/usb/core/buffer.c
+!Idrivers/usb/core/buffer.c
     </chapter>
 
     <chapter>
